src/assistant.js

```diff
@@ -409,7 +409,14 @@ async function getLastSelectedTaskType() {
 		},
 	}
 	const url = generateUrl('/apps/assistant/config')
-	return axios.get(url, req)
+	return axios.get(url, req).data.catch(error => {
```

Comment:
This won't work, I think. `axios.get` returns a promise
```suggestion
	return axios.get(url, req).catch(error => {
```